macrosynergy/dataquery/api.py

```diff
@@ -109,7 +109,7 @@ def server_retry(response: dict, select: str):
         try:
             response[select]
         except KeyError:
-            print(f"{response['errors'][0]['message']} - will try a different server.")
+            print(f"{response['errors'][0]['message']} no instruments in response - will try a different server.")
```

Comment:
I would suggest we write it as:
```python
print(f"Key {select} not found in response: {response} --- retry download.")
```

Reply:
Transferring to an issue along with setting the delay parameter's default value to zero. Will close PR. Housekeeping.